pkg/controller/cincinnati/cincinnati_controller.go

```diff
@@ -183,6 +188,72 @@ func handleErr(reqLogger logr.Logger, status *cv1alpha1.CincinnatiStatus, reason
 	reqLogger.Error(e, reason)
 }
 
+func (r *ReconcileCincinnati) ensureAdditionalTrustedCA(ctx context.Context, reqLogger logr.Logger, instance *cv1alpha1.Cincinnati, resources *kubeResources) error {
+	defaultOpenShiftConfigNS := "openshift-config"
+
+	// Check if the Cluster is aware of a registry requiring an
+	// AdditionalTrustedCA
+	regconf := &apicfgv1.RegistrySources{}
+	image := &apicfgv1.Image{
```

Comment:
`import "k8s.io/apimachinery/pkg/types"`
```suggestion
	image := &apicfgv1.Image{}
	err = r.client.Get(ctx, types.NamespacedName{Name: defaults.ImageConfigName}, image)
```

Reply:
Maybe this didn't work for you before? but `ObjectKeyFromObject` returns an [ObjectKey](https://godoc.org/sigs.k8s.io/controller-runtime/pkg/client#ObjectKey) which is a NamespacedName.

Reply:
Should we have a watch on these?

Reply:
yes, we mainly need to watch the openshift-config configmap since that's our source

Reply:
What configmap is being watched, I'm not certain of.  I was wondering that earlier too: https://github.com/openshift/cincinnati-operator/pull/11#issuecomment-614260225

Reply:
I suggested new Watches just now in my review comment below.